code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -224,6 +234,13 @@
 		playsound(M, "sparks", 50, 1)
 	..()
 
+/datum/reagent/teslium/on_mob_end_metabolize(mob/living/L)
+	. = ..()
+	if(empremoval)
+		var/datum/component/empprotection/empproof = L.GetExactComponent(/datum/component/empprotection)
+		if(empproof)
+			empproof.Destroy()	
```

Comment:
```suggestion
			L.RemoveComponent(/datum/component/empprotection)
```

From what I remember I think Destroy isn't recommended for getting rid of components so that is the way I've seen it done instead.

Reply:
you'd think that, but empprotection is fucked, if you use removecomponent it doesn't actually remove it, only destroy works.